package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "@woocommerce/block-library",
 	"title": "WooCommerce Blocks",
 	"author": "Automattic",
-	"version": "2.9.0-dev",
+	"version": "2.9.0",
```

Comment:
Just noting, these version changes will happen automatically as a part of the `npm run deploy` command.

Reply:
Ah, I had no idea - I'll look at clarifying this in the release doc as follow up.

Reply:
I'm guessing it's this line that was what tripped you up:

> Make any other changes to readme as needed - e.g. support versions changing, new blocks.

"support versions changing" is things like this in the readme.txt:

```
Requires at least: 5.2
Tested up to: 5.4
Requires PHP: 5.6
```
... and this in `woo-gutenberg-products-block.php`

```
* Requires at least: 5.2
 * Requires PHP: 5.6
 * WC requires at least: 4.0
 * WC tested up to: 4.2
```

Later in the release docs is this:

> - Prepare tagged release on github $ npm run deploy.
>      - Note: the script automatically updates version numbers (commits on your behalf).

I agree, it probably could be clarified a bit, I'm just including all this info to maybe help you with how to clarify maybe?

Reply:
Yes that helps a lot - thanks! I think now we have the release PR we can streamline the markdown doc a little bit.